<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MdiEditor</class>
 <widget class="QMainWindow" name="MdiEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MdiEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionCloseAll"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionFont"/>
   <addaction name="separator"/>
   <addaction name="actionMdi"/>
   <addaction name="actionCascade"/>
   <addaction name="actionTile"/>
   <addaction name="separator"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_editor/xinjianwenjian.svg</normaloff>:/icon_editor/xinjianwenjian.svg</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_editor/dakaiwenjian.svg</normaloff>:/icon_editor/dakaiwenjian.svg</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_editor/baocun.svg</normaloff>:/icon_editor/baocun.svg</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionCloseAll">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_editor/guanbisuoyou.svg</normaloff>:/icon_editor/guanbisuoyou.svg</iconset>
   </property>
   <property name="text">
    <string>关闭全部</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_editor/cut.svg</normaloff>:/icon_editor/cut.svg</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_editor/fuzhi.svg</normaloff>:/icon_editor/fuzhi.svg</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_editor/paste.svg</normaloff>:/icon_editor/paste.svg</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_editor/ziti.svg</normaloff>:/icon_editor/ziti.svg</iconset>
   </property>
   <property name="text">
    <string>设置字体</string>
   </property>
  </action>
  <action name="actionMdi">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_editor/tabs.svg</normaloff>:/icon_editor/tabs.svg</iconset>
   </property>
   <property name="text">
    <string>MDI模式</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_editor/jilianxuanze.svg</normaloff>:/icon_editor/jilianxuanze.svg</iconset>
   </property>
   <property name="text">
    <string>级联模式</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_editor/pingpupailie.svg</normaloff>:/icon_editor/pingpupailie.svg</iconset>
   </property>
   <property name="text">
    <string>平铺模式</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon_editor/guanbi_2.svg</normaloff>:/icon_editor/guanbi_2.svg</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MdiEditor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
